### PR TITLE
attendees seeding script

### PR DESCRIPTION
**Pull request by emkwak**

### CODE REVIEW

File: seed.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+const faker = require('faker');
+const _ = require('lodash');
+
+// {
+//   memberId: String('m' + Number)
+//   name: String
+//   avatar: String(URL)
+//   thumbnail: String(URL)
+//   eventOrganizer: Boolean
```
Comment (manuelag19) [inline seed.js]: 1 or 2 see line 54

File: seed.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+//   eventId: Number
+//   limit: Boolean
+//   setLimit: Number / NULL
+//   attendees: Array
+//   waitlist: Array
+// }
+
+
+
+// Attendees
```
Comment (manuelag19) [inline seed.js]: Comment might be from previous commit?

File: seed.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+let members = [];
+
+for (let i = 0; i < 499; i++) {
```
Comment (manuelag19) [inline seed.js]: i should go cover 500 members from 0 to 499, so should be i < 500, otherwise, will only generate 499 member Ids

File: seed.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  const memberId = `m${i}`;
+  const name = faker.name.findName();
+  const avatar = 'http://placecorgi.com/100';
+  const thumbnail = 'http://placecorgi.com/50';
+  const favorite = faker.random.boolean();
```
Comment (manuelag19) [inline seed.js]: This one might not be necessary, the favorite or 'star' function is related to the particular user logged in, so you'd probably be saving that in the sate of the component in react.

File: seed.js [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+// Events
+let events = [];
+
+for (let i = 0; i < 100; i++) {
+  const eventId = i;
+  const limit = faker.random.boolean();
+  const setLimit = limit ? faker.random.number({ min: 20, max: 100 }) : null;
+  const randomNum = faker.random.number({ min: 1, max: 100 });
+  const attendees = _.sampleSize(members, randomNum);
```
Comment (manuelag19) [inline seed.js]: Looks good!

File: seed.js (hunk 1 above)
Comment (emkwak) [reply seed.js]: @manuelag19 I made changes to this

File: seed.js (hunk 4 above)
Comment (emkwak) [reply seed.js]: @manuelag19 users might have favorited an event prior to the event. I can toggle the boolean  in state if the logged in user decides to favorite or unfavorite an event.

**Pull request merged**